### PR TITLE
#166240946 User should be able to create an account

### PR DESCRIPTION
**Pull request by eliemugenzi**
### What does this PR do?
- Create a user interface which helps users create an account
### Description of a task to be completed
- Creating a UI for users to sign up
- Make a signup remote request to the server
### How should this be manually tested?
- Clone this repo
- Run `npm install` or `yarn`
- Run the project using `yarn dev` or `npm run dev`
- Visit `http://localhost:8000/auth/signup`

### Relevant PT story
-  [#166240946](https://www.pivotaltracker.com/story/show/166240946)

### CODE REVIEW

**Review by deschantkn — COMMENTED** [review]
Great job overall @eliemugenzi . However, I don't see any input validation on the signup form. It is best practice to also validate data on the frontend.

**Comment by eliemugenzi** [comment]
> Great job overall @eliemugenzi . However, I don't see any input validation on the signup form. It is best practice to also validate data on the frontend.

@deschantkn , thanks. I AM DONE!

**Review by deschantkn — APPROVED** [review]

File: __tests__/components/signup.test.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    });
+    it('Should change confirmPassword state', () => {
+      const confirm = signup.find('input[name="confirm"]');
+      confirm.simulate('change', {
+        target: { value: '123456' },
+      });
+      expect(signup.state('confirmPassword')).toEqual('123456');
+    });
+    it('Should change DOB state', () => {
+      // console.log(signup.instance().props.createAccount);
```
Comment (Diama1) [inline __tests__/components/signup.test.js]: it would be better if you remove these console comments.

Comment (eliemugenzi) [reply __tests__/components/signup.test.js]: Thanks @Diama1 , RESOLVED!

**Review by Diama1 — APPROVED** [review]

**Review by rukundoeric — APPROVED** [review]

**Pull request merged**